### ISSUE TITLE
Is it possible to make it run without node (in browser)?

### ISSUE THREAD

**Issue by f-person** [zyrouge/genius-lyrics#24]
Hi! I'm building a browser extension which displays lyrics from Genius in another website. When I try to import this library I get the following error: `TypeError: $bGKbb.versions.node is undefined`. It will be nice to make this module run in browsers too.

**Comment by zyrouge** [comment]
It's not possible (considering cors). And that error doesn't seem to come from this module's code. I should be possible to use this on browsers afaik.

**Comment by f-person** [comment]
> And that error doesn't seem to come from this module's code

It happens whenever I try to `require("genius-lyrics")`. 

> considering cors

It might be possible to use in a background script (not sure, however, need to check)

**Comment by f-person** [comment]
I'm using https://github.com/fregante/browser-extension-template BTW

**Comment by f-person** [comment]
> It might be possible to use in a background script

Yup, it's allowed in Chromium (quoting https://www.chromium.org/Home/chromium-security/extension-content-script-fetches):
> tl;dr: To improve security, cross-origin fetches will soon be disallowed from content scripts in Chrome Extensions.  Such requests can be made from extension background pages instead, and relayed to content scripts when needed.

**Comment by zyrouge** [comment]
@f-person Do you know where the error comes from? It might be from some dependencies this package is relying on.

**Comment by f-person** [comment]
Unfortunately, no

**Comment by zyrouge** [comment]
> Unfortunately, no

It should be from somewhere else, there is no such line in this package accessing those properties.

**Comment by f-person** [comment]
Yup, I grepped it too :(

**Comment by TrashUwU** [comment]
@zyrouge hi

**Comment by zyrouge** [comment]
I guess this is the end of the conversation. I'll close this issue since it's stale.
